README.md

```diff
@@ -151,7 +151,9 @@ SwiftClient offers the following requests:
 * `bulk_delete(entries, options = {}) # => entries`
 * `post_head(object_name, container_name, _headers = {}, options = {}) # => HTTParty::Response`
 
-By default, the client instructs the Swift server to return JSON via an HTTP Accept header; to disable this pass `:json => false` in `options`. The rest of the `options` are passed directly to the internal [HTTParty](https://rubygems.org/gems/httparty) client.
+By default, the client instructs the endpoint to return JSON via an HTTP Accept header; to specify another format pass, e.g., `:format => "plain"` in `query` ("xml" is also [supported in the Object Storage API](https://docs.openstack.org/api-ref/object-store/index.html?expanded=show-container-details-and-list-objects-detail#show-container-details-and-list-objects)).
```

Comment:
```suggestion
By default, the client instructs the endpoint to return JSON via an HTTP Accept header. To specify another format pass, e.g., `:query => { :format => "plain" }` via the `options` parameter. Other formats, like `"xml"` are supported as well. Checkout [Object Storage API](https://docs.openstack.org/api-ref/object-store/index.html?expanded=show-container-details-and-list-objects-detail#show-container-details-and-list-objects) for more details.
```

Reply:
besides :+1:, but will be a breaking change.

Reply:
Are you sure about this? A signature for example is,

```ruby
  def get_objects(container_name, query = {}, options = {})
```

(Those methods package `query` into `options for the private `#request`)

How do you feel about the breaking change? I figured it hadn't been "out there" very long but happy to add it back...

Reply:
true. comment updated. I'm basically ok with the breaking change as the proposed change seems to be the stream-line way to me as well.